### ISSUE TITLE
Add buttons in the editor for moving items up and down within a list

### ISSUE THREAD

**Issue by ExpertCoder14** [tryashtar/nbt-studio#14]
Currently, the only way to reorder list items is by dragging and dropping them. However, for those who are particularly unskilled with a mouse, such as disabled, young, or elderly, the only way to reorder list items is by deleting all list items up to the point where the order needs to be changed and then adding them back in the correct order.

To improve accessibility, buttons should be added to the toolbar and/or edit menu to move items up and down within list tags.

**Comment by tryashtar** [comment]
Yep I have actually been planning this, specifically with a shortcut alt+up/down. I tried implementing it reusing drag/drop code and ran into an annoying problem though. So this is on the to-do list but may take some time

**Comment by ExpertCoder14** [comment]
Thanks for that, but I still feel like actual buttons, if not in the toolbar then in the edit menu, would be preferred, because if you add the keyboard shortcut, not everyone is going to actually notice it. If there was a button in the edit menu like "Move Up: Alt+Up Arrow" that would be a good way of letting people know.

**Comment by tryashtar** [comment]
Yeah I'll be sure to include those as well
